src/panels/lovelace/entity-rows/hui-media-player-entity-row.ts

```diff
@@ -71,6 +74,8 @@ class HuiMediaPlayerEntityRow extends LitElement implements LovelaceRow {
       return html``;
     }
 
+    console.log("narrow: " + this._narrow + " veryNarrow: " + this._veryNarrow);
```

Comment:
🔝

Reply:
?

Reply:
We should not commit console log.

Reply:
Sure, but that was not in the most recent changes when you reviewed. I removed that 4 days ago

Reply:
Oh oops, weird it showed those to me when I clicked the notification.